torch/testing/_internal/common_optimizers.py

```diff
@@ -1256,6 +1245,16 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
         ),
         optim_error_inputs_func=optim_error_inputs_func_adam,
         supported_impls=("foreach", "differentiable", "fused"),
+        decorators=(
+            # Expected floating point error between fused and compiled forloop
+            DecorateInfo(
+                toleranceOverride({torch.float64: tol(atol=4.5e-7, rtol=2.2e-6)}),
+                "TestOptimRenewed",
+                "test_fused_matches_forloop",
+                active_if=lambda kwargs: TEST_WITH_TORCHDYNAMO
```

Comment:
```suggestion
                active_if=TEST_WITH_TORCHDYNAMO
```
it takes a boolean too!

Reply:
What do you mean? I used the lambda because I check the dtype too..